miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebChromeClient.kt

```diff
@@ -163,4 +164,21 @@ internal class MiniAppWebChromeClient(
         if (customView != null)
             onHideCustomView()
     }
+
+    @Suppress("TooGenericExceptionCaught", "SwallowedException")
+    override fun onShowFileChooser(
+        webView: WebView?,
+        filePathCallback: ValueCallback<Array<Uri>>?,
+        fileChooserParams: FileChooserParams?
+    ): Boolean {
+        try {
+            miniAppFilePicker?.requestFile(filePathCallback) { requestCode ->
+                val intent = fileChooserParams?.createIntent()
+                (context as Activity).startActivityForResult(intent, requestCode)
+            }
+        } catch (e: Exception) {
+            throw (MiniAppFilePickingException(e.message.toString()))
```

Comment:
Is there any chance that we could get this exception?

Reply:
One chance is `ActivityNotFoundException`.